tests/pipeline/test_runner.py

```diff
@@ -58,6 +73,8 @@ def test_docker_runner_from_pipeline():
                 "--remove-orphans",
             ],
             env=dict(os.environ, DOCKER_DEFAULT_PLATFORM="linux/amd64"),
+            capture_output=True,
+            encoding="utf8",
         )
 
 
```

Comment:
A test that checks the ` PipelineRunError` error might be nice here, we do this for other parts of the codes aswel:
https://github.com/ml6team/fondant/blob/main/tests/test_explorer.py#L164

Reply:
Added tests